src/mbgl/map/transform.cpp

```diff
@@ -55,67 +60,69 @@ bool Transform::resize(const std::array<uint16_t, 2> size) {
 
 #pragma mark - Position
 
-void Transform::jumpTo(const CameraOptions options) {
-    CameraOptions jumpOptions = options;
-    jumpOptions.duration.reset();
-    easeTo(jumpOptions);
+void Transform::jumpTo(CameraOptions& options) {
+    options.duration.reset();
```

Comment:
This line mutates the CameraOptions for local purposes. There are several instances of this in Map and Transform, which is why the CameraOptions was being passed in by value.